PEERS_GUIDELINES.md

```diff
@@ -0,0 +1,180 @@
+  team, which consists of:
+  - [@hmatrjp](https://github.com/hmatrjp)
+  - [@potappo](https://github.com/potappo)
+  - [@dynamis](https://github.com/dynamis)
+  - [@kenji-yamasaki](https://github.com/kenji-yamasaki)
+  - [@mfuji09](https://github.com/mfuji09)
+- Korea (`ko`) content — the
+[@yari-content-ko](https://github.com/orgs/mdn/teams/yari-content-ko)
+  team, which consists of:
+  - [@yechoi42](https://github.com/yechoi42)
+  - [@cos18](https://github.com/cos18)
+  - [@GwangYeol-Im](https://github.com/GwangYeol-Im)
+  - [@pje1740](https://github.com/pje1740)
+  - [@nKiNk](https://github.com/nKiNk)
+  - [@yujo11](https://github.com/yujo11)
+  - [@hochan222](https://github.com/hochan222)
+- Russian (`ru`) content — the
+[@yari-content-ru](https://github.com/orgs/mdn/teams/yari-content-ru)
+  team, which consists of:
+  - [@armanpwnz](https://github.com/armanpwnz)
+  - [@captainspring](https://github.com/captainspring)
+  - [@mpstv](https://github.com/mpstv)
+  - [@myshov](https://github.com/myshov)
+  - [@Saionaro](https://github.com/Saionaro)
+  - [@sashasushko](https://github.com/sashasushko)
+  - [@lex111](https://github.com/lex111)
+- Chinese (`zh-CN` and `zh-TW`) content — the
+[@yari-content-zh](https://github.com/orgs/mdn/teams/yari-content-zh)
+  team, which consists of:
+  - [@t7yang](https://github.com/t7yang)
+  - [@dibery](https://github.com/dibery)
+  - [@irvin](https://github.com/irvin)
+
+---
+
+## Synchronization with the en-US document structure
+
+Periodically we synchronize all the localized document tree structures with the
+`en-US` tree structure (English slugs only), to make the documentation easier
+to manage. When documents are removed from `en-US` it results in `orphaned`
+documents within a sub-directory of each locale folder.
+
+Active locale maintenance teams are invited to spend some time exploring the
+orphaned documents, to see whether any of this work is worth keeping (either
+adding to, or merging with an existing document in, the main tree), or whether
+it can just be deleted.
+
+### Periodic synchronization updates
+
+We run a GitHub Action every day to update the localized documentation and keep
+it in sync with the `en-US` tree structure, for example if documents are deleted
+for, or moved to a different location, in the tree.
+
+When a synchronization occurs maintenance teams are given two weeks to decide
+ what to do with the affected documents in their locales to keep things in sync.
+
+Note: Conflicting docs are often created during the sync operation when `en-US`
+documents get merged — for example if `Foo/Bar` becomes just a section inside
+`Foo`, and we redirect `Foo/Bar` to `Foo#Bar`. This will result in a conflict
+as the sync job tries to move the translated `Foo/Bar` to `Foo` according to the
+redirect, but `Foo` already exists.
+
+---
+
+## Activating a locale
+
+To activate a locale you must apply in writing with the following information:
```

Comment:
Imagining myself in the shoes of someone who might be interested in starting or reviving a not-active locale, by first question is: What does that mean? To "apply in writing"? Do I send a paper-letter? Do I email someone? Do I fill in a form? Do I file an issue?

Reply:
There's a contact bit below, so I just removed 'in writing' - I'm pretty sure requests will come in as issues or in matrix, so it's just a case of reaching out directly to _them_

Doing this was just to get something in writing to the es locale - I wonder at all if it will be used again honestly